assets/js/components/HostDetails/HostClusterAgentIpSummary.jsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import ListView from '@components/ListView';
+import HealthIcon from '@components/Health/HealthIcon';
+import Tooltip from '@components/Tooltip';
+import ClusterLink from '@components/ClusterLink';
+
+import { chunk } from 'lodash';
+
+function HostClusterAgentIpSummary({ agentVersion, cluster, ipAddresses }) {
+  const prepareTooltipContent = (ipList) => {
+    const formattedIpList = chunk(ipList, 2);
+    const preparedTooltipContent = formattedIpList.map((ipPair) => {
```

Comment:
Or even without the comma thing, if we add more gap

Reply:
Yeah it is complicated but i tried to hit @jagabomb  design mockup :)
You propose it like this correct? 
![image](https://github.com/trento-project/web/assets/54111255/7e2a6e77-8eda-4b5c-9156-4df8893cdd1f)
Instead of: 
![image](https://github.com/trento-project/web/assets/54111255/18b6ed1b-440c-45ff-b113-83e9252d99f8)

For me both solutions are fine. 
@jagabomb  what do you prefer visually ?

Reply:
Well, my proposal was just pseudo code. Most probably you can achieve the same output just with some styling

Reply:
I will try out :D

Reply:
Actually with a fresh look on it i even prefer @arbulu89  option, it is much easier to read as the first implementation @jagabomb what do you think ? :D